### ISSUE TITLE
Uncrustify needs two passes for one liner classes

### ISSUE THREAD

**Issue by mihaipopescu** [uncrustify/uncrustify#1258]
```
namespace Unity.IL2CPP.IntegrationTests.ILTests.Tests
{
	public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase { public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) { } }
	public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase { public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) { } }
	public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase { public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) { } }
	public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase { public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) { } }
}
```

Expected versus output diff:
---
```diff
diff --git "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60009-UNI-9917.cs" "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60009-UNI-9917.cs"
index c858486..e05bc2d 100644
--- "a/C:\\work\\external\\uncrustify\\tests\\output\\staging\\60009-UNI-9917.cs"
+++ "b/C:\\work\\external\\uncrustify\\tests\\results\\staging\\60009-UNI-9917.cs"
@@ -1,65 +1,73 @@
 // 1. Formatting the first produces the second.
 
 namespace Unity.IL2CPP.IntegrationTests.ILTests.Tests
 {
-    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue)
+        {
+        }
     }
-    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S)
+        {
+        }
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse)
+        {
+        }
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S)
+        {
+        }
     }
 }
 
 // 2. And formatting this produces the third.
 
 namespace Unity.IL2CPP.IntegrationTests.ILTests.Tests
 {
-    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) {}
     }
 }
 
 // 3. This doesn't changes when formatted.
 
 namespace Unity.IL2CPP.IntegrationTests.ILTests.Tests
 {
-    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueOpcodeTest() : base(OpCodes.Brtrue) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrTrueSOpcodeTest() : base(OpCodes.Brtrue_S) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseOpcodeTest() : base(OpCodes.Brfalse) {}
     }
-    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase 
-    { 
-        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) { } 
+    public class GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest : BoxTestsBase
+    {
+        public GlobalsWithBoxOptimizationAndBrFalseSOpcodeTest() : base(OpCodes.Brfalse_S) {}
     }
 }
```

Debug support files:
---
[60009-UNI-9917.cs](https://gist.githubusercontent.com/mihaipopescu/8e19bc03e81a173d06e2ac2ceeef4727/raw/c8584868ef8270d04dcb4171d8d55f891bf3bdee/60009-UNI-9917.cs)
[60009-UNI-9917.cs.log](https://gist.githubusercontent.com/mihaipopescu/8e19bc03e81a173d06e2ac2ceeef4727/raw/a31470dec92b0f1ddbfff26278d279211937bc15/60009-UNI-9917.cs.log)
[60009-UNI-9917.cs.unc](https://gist.githubusercontent.com/mihaipopescu/8e19bc03e81a173d06e2ac2ceeef4727/raw/7068676e29a3f98111c03961a477555ac285cb7e/60009-UNI-9917.cs.unc)
[Uncrustify.CSharp.cfg](https://gist.githubusercontent.com/mihaipopescu/8e19bc03e81a173d06e2ac2ceeef4727/raw/1d064c1fabb76ce328dd015ed40023207de20ca1/Uncrustify.CSharp.cfg)
[UNI-9917.cs](https://gist.githubusercontent.com/mihaipopescu/8e19bc03e81a173d06e2ac2ceeef4727/raw/5bd9dc6456c240c0541438c7a66addef4d39771e/UNI-9917.cs)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by CDanU** [comment]
Here
```sql
nl_class_brace                  = force
```
seems to overpower the one liner options completely, i.e. I don't get the one liner back into one line again after the initial formatting.

**Comment by gmaurel** [comment]
Fixed at 9a3e20af24103a3dacefd9297a89c7d2d4075918, on Jan  3 08:23:12 2018
